op-chain-ops/genesis/genesis.go

```diff
@@ -53,6 +53,7 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		CancunBlock:                   nil,
 		TerminalTotalDifficulty:       big.NewInt(0),
 		TerminalTotalDifficultyPassed: true,
+		BedrockBlock:                  new(big.Int).SetUint64(uint64(config.L2GenesisBlockNumber)),
```

Comment:
Hmm bedrock block isn't under `Optimism`?

Reply:
no, it's not. Easier to get to & it matches the other forks better.

Reply:
TBH I like it better to group optimism configurables, it'll make things more maintainable as diff. The consistency with other fork block numbers is nice too though.